packages/dbml-core/src/parse/mysql/parser.pegjs

```diff
@@ -190,7 +190,7 @@ UniqueSyntax
 // "KEY is normally a synonym for INDEX".
 IndexInLineSyntax = _ unique:index_in_line
 	_ name:name? _ type1:index_type? _
-	"(" _ columns:IndexColumnValues _")" IndexOption? type2:index_type?
+	"(" _ columns:IndexColumnValues _")" type2:(_ IndexOption)? type3:(_ index_type)?
```

Comment:
since `IndexOption` already includes `index_type`, we don't need type3 here. This is also more accurate with official MySQL syntax of `create_definition`
```sql
[CONSTRAINT [symbol]] UNIQUE [INDEX | KEY]
      [index_name] [index_type] (key_part,...)
      [index_option] ...
```
(ref: https://dev.mysql.com/doc/refman/8.0/en/create-table.html)
```suggestion
	"(" _ columns:IndexColumnValues _")" type2:(_ IndexOption)?
```